modular_skyrat/modules/oversized/code/oversized_quirk.dm

```diff
@@ -32,6 +40,7 @@
 	human_holder.dna.species.punchdamagelow -= OVERSIZED_HARM_DAMAGE_BONUS
 	human_holder.dna.species.punchdamagehigh -= OVERSIZED_HARM_DAMAGE_BONUS
 	human_holder.blood_volume_normal = BLOOD_VOLUME_NORMAL
+	human_holder.physiology.hunger_mod /= 1.5
```

Comment:
dividing when multiplication is doable is against code standards

Reply:
This is how it's done for everything else, tho.

Reply:
Doing that would mean we'd have `*= 2/3` which is quite honestly even less pretty. That being said, this 1.5 should be a define, `OVERSIZED_HUNGER_MODIFIER` or something like that.

Reply:
I'm actually yelling.